src/components/student/ScheduleView.js

```diff
@@ -9,14 +14,130 @@ import React, {useState} from 'react';
 // issue a DELETE with URL /enrollment/{enrollmentId}
 
 const ScheduleView = (props) => {
-    
-   
-    return(
-        < > 
-            <h3>Not implemented</h3>
-        </ >
-    );
-
-}
+
+    const headers = ['SecNo', 'CourseId', 'SecId',  'Year', 'Semester', 'Building', 'Room', 'Times', '', ''];
+
+    const [search, setSearch] = useState({
+        scheduleId: '',
+        year: '',
+        semester: ''
+    });
+    const [message, setMessage] = useState('');
+    const [schedules, setSchedules] = useState([]);
+
+    const fetchSections = async () => {
+            if (search.courseId==='' || search.year==='' || search.semester==='' ) {
+                setMessage("Enter search parameters");
+            } else {
+              try {
+                const response = await fetch(`${SERVER_URL}/courses/${search.courseId}/sections?year=${search.year}&semester=${search.semester}`);
```

Comment:
We definitely need to change this. We should be fetching `/enrollments?studentId=${studentId}&year=${search.year}&semester=${search.semester}`. After we change this we can also change the data displayed in the table to match Enrollment DTO. For `studentId` you can just hard-code `studentId=3` for now at the top of the component. See the comments at the top of the file.

Reply:
Corrected the URL in ScheduleView to ``${SERVER_URL}/enrollments?year=${search.year}&semester=${search.semester}&studentId=3``